app/services/solidus_bolt/accounts/detail_service.rb

```diff
@@ -11,7 +11,9 @@ def initialize(access_token:)
       end
 
       def call
-        detail
+        response = detail
+        update_user(response)
```

Comment:
I don't think this is the right place to update the user info.
What do you think to update the payment_sources before the payment step during the checkout?

Reply:
removed